README.md

```diff
@@ -743,7 +743,7 @@ To restore from a backup after some disaster, just put that secrets back before
 
 ```bash
 kubectl apply -f main.key
-kubectl delete pod -n kube-system -l name=sealed-secrets-controller
+kubectl delete pod -n kube-system -l app.kubernetes.io/name=sealed-secrets
```

Comment:
this commands will work only for the chart, we need to keep the other one for the controller.yaml. Could help us to include the commands for the controller.yaml (keeping the original as we have) and including the new commands for the charts (as you are doing right now), please?
example:

```
* Deploying Sealed Secrets using Helm:
COMMANDS

* Deploying Sealed Secrets using the controller.yaml
COMMANDS
```

Reply:
Thanks for picking that up. Fixed.